packages/cli/src/commands/reserve/transfergold.test.ts

```diff
@@ -1,33 +1,47 @@
+import { newKitFromWeb3 } from '@celo/contractkit'
+import { GoldTokenWrapper } from '@celo/contractkit/lib/wrappers/GoldTokenWrapper'
 import { testWithGanache } from '@celo/dev-utils/lib/ganache-test'
+import { Address } from '@celo/utils/lib/address'
+import BigNumber from 'bignumber.js'
 import Web3 from 'web3'
 import TransferGold from './transfergold'
 
 process.env.NO_SYNCCHECK = 'true'
 
 testWithGanache('reserve:transfergold cmd', (web3: Web3) => {
+  const transferAmt = new BigNumber(100000)
+  const kit = newKitFromWeb3(web3)
+
+  let accounts: Address[] = []
+  let goldToken: GoldTokenWrapper
+
+  beforeEach(async () => {
+    accounts = await web3.eth.getAccounts()
+    goldToken = await kit.contracts.getGoldToken()
+  })
   test('transferGold fails if spender not passed in', async () => {
-    const accounts = await web3.eth.getAccounts()
     await expect(
       TransferGold.run([
         '--from',
         accounts[0],
         '--value',
-        '1',
+        transferAmt.toString(10),
         '--to',
-        '0x91c987bf62D25945dB517BDAa840A6c661374402',
+        accounts[9],
       ])
     ).rejects.toThrow("Some checks didn't pass!")
   })
   test('can transferGold with multisig option', async () => {
-    const accounts = await web3.eth.getAccounts()
+    const initialBalance = await goldToken.balanceOf(accounts[9])
     await TransferGold.run([
       '--from',
       accounts[0],
       '--value',
-      '1',
+      transferAmt.toString(10),
       '--to',
-      '0x91c987bf62D25945dB517BDAa840A6c661374402',
+      accounts[9],
       '--useMultiSig',
     ])
+    expect(await goldToken.balanceOf(accounts[9])).toEqual(initialBalance)
```

Comment:
Wouldn't we expect this balance to go up by `transferAmt`?

Reply:
ya...i thought that was what I had here...perhaps i removed the addition when debugging and then it passed...which is a bad sign